charts/sync/templates/sync.yaml

```
@@ -0,0 +1,140 @@
+apiVersion: canaries.flanksource.com/v1
+kind: Topology
+metadata:
+  name: {{ .Values.topologyName }}
+spec:
+  icon: sync
+  type: Topology
+  schedule: "@every 2m"
+  components:
+    - name: Check Statuses
+      icon: helm
+      properties:
+        - name: Backlog
```

```suggestion
        - name: Check Status Backlog
```

Will also need for config_analysis and config_changes